pkg/utils/che/che.go

```diff
@@ -0,0 +1,54 @@
+package che
+
+import (
+	olmv1 "github.com/operator-framework/operator-lifecycle-manager/pkg/api/apis/operators/v1"
+	olmv1alpha1 "github.com/operator-framework/operator-lifecycle-manager/pkg/api/apis/operators/v1alpha1"
+
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+//Subscription for eclipse Che operator
+func Subscription(ns string) *olmv1alpha1.Subscription {
+	return &olmv1alpha1.Subscription{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "eclipse-che",
+			Namespace: ns,
+			Labels:    Labels(),
+		},
+		Spec: &olmv1alpha1.SubscriptionSpec{
+			Channel:                "stable",
+			Package:                "eclipse-che",
+			StartingCSV:            "eclipse-che.v7.2.0",
```

Comment:
We will need to parametrize it later (Che/CRW, version, channel, etc). But it's OK to keep them hardcoded for now.

Reply:
yes, the plan it to make it parameterized when needed